website/docusaurus.config-blog.js

```diff
@@ -50,6 +50,26 @@ const config = {
 
           blogSidebarTitle: 'All posts',
           blogSidebarCount: 'ALL',
+          feedOptions: {
+            type: 'rss', // or whichever feed types you prefer
```

Comment:
That seems to be copied and pasted from somewhere, including the comment. I think that also disables Atom and JSON feed.

Reply:
Yes, it disables JSON and Atom feed for now (although the json wasn't enabled before). I still need to modify it for atom, but we need the rss feed now for the website though.

Reply:
Resolved it now. It should work for the 3 feeds 😊

Reply:
https://7dee85a1.ferretdb-blog-dev.pages.dev/feed.json